src/test/java/StringClassTest.java

```diff
@@ -0,0 +1,67 @@
+import org.junit.jupiter.api.*;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
+
+public class StringClassTest {
+
+    @Nested
+    @DisplayName("첫번째 요구사항 클래스")
+    class NumberTest {
+          private String NumbersData = "1,2";
+          private String NumberData = "1";
+          private String [] array;
+
+        @Test
+        @DisplayName(value = "1,2 split으로 분리여부 확인 테스트")
+        public void splitNumbersTest() {
+            array = NumbersData.split(",");
+            assertThat(array).contains("1","2");
+        }
+
+        @Test
+        @DisplayName(value = "1 split후 1만 포함하는지 확인 테스트")
+        public void splitNumberTest() {
+            array = NumberData.split(",");
+            assertThat(array).containsExactlyInAnyOrder("1");
+        }
+
+    }
+
+    @Nested
+    @DisplayName("두번째 요구사항 클래스")
+    class UseSubStringTest {
+        private String require2NumberData = "(1,2)";
+
+        @Test
+        @DisplayName(value = "(1,2) subString사용해서 1,2만 반환 테스트")
+        public void shouldReturnNumberTest() {
+            String result = require2NumberData.substring(1,4);
+            assertThat("1,2").isEqualTo(result);
```

Comment:
```suggestion
            assertThat(result).isEqualTo("1,2");
```

결과와 검증하려는 값의 위치가 바뀌었다.

Reply:
위치 수정 완료했습니다!